azerothcore/azerothcore-wotlk
# fix(Core/Spells): Fixed calculating damage/healing pct auras on aura …

…refresh.

Fixes #8962



## Issues Addressed:

- Closes #8962

## Tests Performed:

- Not tested.

## How to Test the Changes:

summon dummy (.npc add 32667).
cast Shadow Word: Pain (.cast 48125) and see the damage.
cast Shadowform (.cast 15473) and refresh Shadow Word: Pain (.cast 48125). The damage is the same.
now wait for Shadow Word: Pain to end and cast again (.cast 48125) with Shadowform. The damage is increased.


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
